README.md

```diff
@@ -61,6 +61,8 @@ Configuration file: /octocat/personal-website/_config.yml
 
 Don't worry about the GitHub API errors. They are caused because it's run in a dev enviornment and not on GitHub Pages.
 
+You can set a GitHub API Token by setting the JEKYLL_GITHUB_TOKEN to your personal access token.
```

Comment:
Might wanna add backticks around JEKYLL_GITHUB_TOKEN. (Like this: `JEKYLL_GITHUB_TOKEN `)

Reply:
Implemented

Reply:
👍